pkgs/app/src/components/Help/index.tsx

```diff
@@ -1,4 +1,4 @@
-import { SiDiscord, SiGithub } from '@icons-pack/react-simple-icons';
+import { SiDiscord, SiGitHub } from '@icons-pack/react-simple-icons';
```

Comment:
react-simple-icons does not export the icon with this naming.
Hopefully, you can run the test by building the app 
```sh
npm run app
or
turbo build --filter app
```